repository-data/application/src/main/resources/hcm-config/namespaces/hee/event.yaml

```diff
@@ -60,6 +60,57 @@ definitions:
             hipposysedit:primary: false
             hipposysedit:type: String
             hipposysedit:validators: [required]
+        /hipposysedit:nodetype[2]:
```

Comment:
Looks like a duplicate node - could we remove if its redundant ?

Reply:
Removed.